pylint to precommit. add pylintrc

FoamyGuy opened the pull request.

FoamyGuy committed:
pylint to precommit. add pylintrc

hugodahl commented:
Looking at this change since the PR's been open for a while, it seems that this repo doesn't have a `requirements.txt` file, which is tripping up the "Install dependencies" step of the process.

Adding an empty `requirements.txt` file with the following contents to this PR should make checks pass

```requirements.txt
# SPDX-FileCopyrightText: 2021 Adafruit Industries
#
# SPDX-License-Identifier: Unlicense
```

hugodahl committed:
Create requirements.txt

Add an empty requirements file to satisfy the "Install dependencies" step of the build actions, as mentioned in [my comment](https://github.com/adafruit/Adafruit_CircuitPython_EthernetManager/pull/1#issuecomment-809841224) on [PR #1](https://github.com/adafruit/Adafruit_CircuitPython_EthernetManager/pull/1e)

FoamyGuy committed:
Merge pull request #1 from hugodahl/patch-1

Create requirements.txt

tannewt commented:
Archiving this.
